test/name_bazaar/smart_contract_tests.cljs

```diff
@@ -71,19 +71,124 @@
+                                                          {:ens.record/label "abc" :ens.record/owner offering}
+                                                          {:from (state/my-address 0)}))))
+                   (is (tx-failed? (<! (auction-offering/bid! ss
+                                                              {:offering/address offering}
+                                                              {:value (web3/to-wei 0.09 :ether)
+                                                               :from (state/my-address 1)}))))
+                   (is (tx-sent? (<! (auction-offering/bid! ss
+                                                            {:offering/address offering}
+                                                            {:value (web3/to-wei 0.1 :ether)
+                                                             :from (state/my-address 1)}))))
+                   (is (tx-failed? (<! (auction-offering/bid! ss
+                                                              {:offering/address offering}
+                                                              {:value (web3/to-wei 0.11 :ether)
+                                                               :from (state/my-address 2)}))))
+                   (is (tx-sent? (<! (auction-offering/bid! ss
+                                                              {:offering/address offering}
+                                                              {:value (web3/to-wei 0.2 :ether)
+                                                               :from (state/my-address 2)}))))
+                   (is (tx-sent? (<! (auction-offering/bid! ss
+                                                            {:offering/address offering}
+                                                            {:value (web3/to-wei 0.33 :ether)
+                                                             :from (state/my-address 3)}))))
+                   (is (= {:auction-offering/min-bid-increase 100000000000000000
+                           :auction-offering/winning-bidder (state/my-address 3)
+                           :auction-offering/bid-count 3}
+                          (select-keys (last (<! (auction-offering/get-auction-offering ss
+                                                                                        offering)))
+
+                                       [:auction-offering/min-bid-increase
+                                        :auction-offering/winning-bidder
+                                        :auction-offering/bid-count])))
+
+                   (is (tx-failed? (<! (auction-offering/finalize! ss
+                                                             {:offering/address offering
+                                                              :offering/transferPrice true}
+                                                             {:from (state/my-address 0)}))))
+                   (<! (timeout 5000))
```

Comment:
there's a method in `cljs-web3.async.evm/increase-time!` where you can fast forward testrpc time. See readme at https://github.com/district0x/cljs-web3 on how to use it